src/lib/viewers/office/OfficeLoader.js

```diff
@@ -29,14 +30,15 @@ class OfficeLoader extends AssetLoader {
     /**
      * @inheritdoc
      */
-    determineViewer(file, disabledViewers = []) {
+    determineViewer(file, disabledViewers = [], viewerOptions = {}) {
         // The Office viewer is disabled when this is a password protected shared link
         const isDisabledDueToPasswordProtectedSharedLink = file.shared_link && file.shared_link.is_password_enabled;
-        // If the user does not have permission to download the file, the file is larger than 5MB, or isDisabledDueToSharedLink is true,
-        // then disable the Office viewer
+        // If the user does not have permission to download the file, the file is larger than Appconf file size or 5MB,
+        // or isDisabledDueToSharedLink is true, then disable the Office viewer
+        const maxFileSize = get(viewerOptions, 'office.excelOnlineAllowedMaxFileSize') || FIVE_MB;
```

Comment:
You can default it to 5MB `get(viewerOptions, 'office.excelOnlineAllowedMaxFileSize', FIVE_MB)`